app/services/dataworks_mappers/sdr.rb

```diff
@@ -0,0 +1,471 @@
+# frozen_string_literal: true
+
+module DataworksMappers
+  # Map fields from SDR cocina records to Dataworks metadata
+  # rubocop:disable Metrics/ClassLength
+  class Sdr < Base
+    include ActiveSupport::NumberHelper
+
+    # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
+    def perform_map
+      {
+        creators:,
+        titles:,
+        publisher:,
+        publication_year:,
+        subjects:,
+        contributors:,
+        descriptions:,
+        dates:,
+        language:,
+        version:,
+        identifiers:,
+        related_identifiers:,
+        sizes:,
+        formats:,
+        rights_list:,
+        funding_references:,
+        related_items:,
+        url:,
+        access:,
+        provider: 'SDR'
+      }.compact_blank
+    end
+    # rubocop:enable Metrics/MethodLength, Metrics/AbcSize
+
+    private
+
+    # Not mapping alternative/subtitles at the moment
+    def titles(element = source.description.title)
+      return [] if element.blank?
+
+      [
+        { title: Cocina::Models::Builders::TitleBuilder.full_title(element).first }
+      ]
+    end
+
+    def identifiers
+      # Always present
+      ids = [
+        { identifier: source.identification.sourceId, identifier_type: 'SourceId' },
```

Comment:
I don't think SourceId should be exposed publicly. It is solely an internal identifier.

Reply:
It isn't public, is it? I thought we might need this to do things like generate stacks URLs or other stuff in the frontend later on. But afaik there's no reason it would be public since we don't map it to anything, right?

Reply:
Removed it since there's no use case right now.